uwh-refbox/src/main.rs

```diff
@@ -140,53 +140,259 @@ fn main() -> std::result::Result<(), Box<dyn std::error::Error>> {
+                        );
+                    }
+                    _ => display.draw(
+                        Font8x15::render_str("ERROR_2")
+                            .stroke(Some(game_color))
+                            .translate(Point::new(64, 2)),
+                    ),
+                }
+                match state.timeout {
+                    TimeoutState::White(_) => {
+                        display.draw(
+                            Font8x15::render_str("WHITE")
+                                .stroke(Some(timeout_color))
+                                .translate(Point::new(76, 2)),
+                        );
+                        display.draw(
+                            Font8x15::render_str("TIMEOUT")
+                                .stroke(Some(timeout_color))
+                                .translate(Point::new(68, 17)),
+                        );
+                        display.draw(
+                            Font16x31::render_str(":59")
+                                .stroke(Some(timeout_color))
+                                .translate(Point::new(132, 2)),
+                        );
+                    }
+                    TimeoutState::Black(_) => {
+                        display.draw(
+                            Font8x15::render_str("BLACK")
+                                .stroke(Some(timeout_color))
+                                .translate(Point::new(76, 2)),
+                        );
+                        display.draw(
+                            Font8x15::render_str("TIMEOUT")
+                                .stroke(Some(timeout_color))
+                                .translate(Point::new(68, 17)),
+                        );
+                        display.draw(
+                            Font16x31::render_str(":59")
+                                .stroke(Some(timeout_color))
+                                .translate(Point::new(132, 2)),
+                        );
+                    }
+                    TimeoutState::Ref(_) => {
+                        display.draw(
+                            Font11x25::render_str("REF TIMEOUT")
+                                .stroke(Some(game_color))
+                                .translate(Point::new(68, 4)),
+                        );
+                    }
+                    TimeoutState::PenaltyShot(_) => {
+                        display.draw(
+                            Font11x25::render_str("PENALTY")
+                                .stroke(Some(timeout_color))
+                                .translate(Point::new(65, 4)),
+                        );
+                        display.draw(
+                            Font11x25::render_str("SHOT")
+                                .stroke(Some(timeout_color))
+                                .translate(Point::new(149, 4)),
+                        );
+                    }
+                    _ => display.draw(
```

Comment:
No default, just have the `None` case do nothing

Reply:
ok